CONTRIBUTING.md

```diff
@@ -10,10 +10,9 @@ development dependencies by running
 
     $ pip install -e ".[all]"
 
-Verify that all existing tests pass by executing the test suite via
-[nose2][nose2]:
+Verify that all existing tests pass by executing the test suite with:
 
-    $ nose2 tests
+    $ pytest -v --cov-report= --cov=src/pymanopt tests
```

Comment:
@NicolasBoumal 
Codacy seems to have a [problem with this line](https://app.codacy.com/gh/pymanopt/pymanopt/pull-requests/279/issues#issue-7201c644867e40ed8eafa57fce88f5b9) and wants to remove the '$' sign.
 
There are similar issues on other lines in the same file but did not complain because we did not modify them. I think we still should

Reply:
The other issues are still mentioned [in the global analysis](https://app.codacy.com/gh/pymanopt/pymanopt/issues/current).